perfdash/metrics-downloader.go

```diff
@@ -182,13 +175,24 @@ func getBuildData(result JobToCategoryData, prefix string, category string, labe
 	return result[prefix][category][label]
 }
 
-type bucketUtil struct {
+// MetricsBucket is the interface that fetches data from a storage service.
+type MetricsBucket interface {
+	GetBuildNumbers(job string) ([]int, error)
+	ListFilesInBuild(job string, buildNumber int, prefix string) ([]string, error)
+	GetFile(job string, buildNumber int, path string) ([]byte, error)
+}
+
+var _ MetricsBucket = &GCSMetricsBucket{}
```

Comment:
No need for this check, as it's already verified by `NewGCSMetricsBucket`'s return type :)